nixos/tests/all-tests.nix

```
@@ -516,6 +516,7 @@ in {
   lxd-image-server = handleTest ./lxd-image-server.nix {};
   #logstash = handleTest ./logstash.nix {};
   lomiri = handleTest ./lomiri.nix {};
+  lomiri-filemanager-app = handleTest ./lomiri-filemanager-app.nix {};
```

Can you please use `runTest` instead?
```suggestion
  lomiri-filemanager-app = runTest ./lomiri-filemanager-app.nix;
```

---

Neat, TIL.